### ISSUE TITLE
Make tabs with long contents floating on the top

### ISSUE THREAD

**Issue by astroHaoPeng** [theme-next/hexo-theme-next#958]
Is it possible for the tabs to float at the top of the screen, while people scroll down a long tab.

See [my page](https://spaceresearch.top/blog/cv/#6-2-restricted-three-body-problem) for example. Basically I was using tabs as a subsubsection to organize my contents. Since the context in one tab is usually long, I hope the tabs will float on the top of the screen, so that people will not forget to check other tabs.

Please feel free to refuse and close the issue. Thanks.

**Comment by jiangtj** [comment]
Good idea. But I think using header `# ## ###` maybe better, if you want them all to be seen. And they can also be skipped by clicking on the sidebar toc.

BTW, I don't know how to implement this function. It seems a bit difficult. So waiting for someone to be interested in it.

**Comment by stevenjoezhang** [comment]
See https://github.com/next-theme/hexo-theme-next/pull/361

@astroHaoPeng Can you help review this pull request? Thanks
